app/src/test/resources/robolectric.properties

```diff
@@ -0,0 +1,3 @@
+packageName=lt.vilnius.tvarkau
+sdk=23
```

Comment:
Not supported by Robolectric: https://github.com/robolectric/robolectric/issues/2554